src/components/backend-ai-webui.ts

```diff
@@ -1512,9 +1541,12 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
                       graphic="icon"
                       ?selected="${this._page === 'agent-summary'}"
                       @click="${() => this._moveTo('/agent-summary')}"
-                      ?disabled="${this.blockedMenuItem.includes(
+                      ?disabled="${this.inactiveMenuItem.includes(
                         'agent-summary',
                       )}"
+                      class="${this.blockedMenuItem.includes('agent-summary')
+                        ? 'hidden'
+                        : ''}"
                     >
```

Comment:
There was an extra bracket (`>`) in L1551, so I removed it. Except that, It works fine.

Reply:
### Screenshot(s)
<img width="246" alt="Screenshot 2023-11-24 at 5 24 37 PM" src="https://github.com/lablup/backend.ai-webui/assets/46954439/b846827d-2f70-49b2-a38a-313f759fa460">